README.md

```diff
@@ -1,3 +1,7 @@
 # winston
 
 One day, this will be a real README. For now, see [`INSTALL.md`](INSTALL.md).
+
+Possible problems:
```

Comment:
I think we probably want this to be "known issues" or something, if we even want to include it.